server/src/main/java/org/elasticsearch/action/admin/indices/analyze/TransportAnalyzeAction.java

```diff
@@ -488,7 +480,7 @@ private void analyze(TokenStream stream, Analyzer analyzer, String field, Set<St
         }
 
         private AnalyzeAction.AnalyzeToken[] getArrayTokens() {
-            return tokens.toArray(new AnalyzeAction.AnalyzeToken[tokens.size()]);
+            return tokens.toArray(new AnalyzeAction.AnalyzeToken[0]);
```

Comment:
Curious to learn why tokens.size() isn't an advantage to initialize the new array with the right size here?

Reply:
The short version is: IntelliJ marks it as a warning, so I changed it :)
The long version is here: https://shipilev.net/blog/2016/arrays-wisdom-ancients/

Reply:
After quickly browsing the later I'll just go with the short explanation ;-) Thanks.